kotlinpoet/src/test/java/com/squareup/kotlinpoet/FileSpecTest.kt

```diff
@@ -1131,4 +1133,27 @@ class FileSpecTest {
       |""".trimMargin()
     )
   }
+
+  @Test fun defaultImports() {
+    val spec = FileSpec.scriptBuilder("Taco")
+      .addProperty(PropertySpec.builder("prop0", STRING.copy(nullable = true)).initializer("null").build())
+      .addProperty(PropertySpec.builder("prop1", INT.copy(nullable = true)).initializer("null").build())
+      .addProperty(PropertySpec.builder("prop2", typeNameOf<Map<String, Any>?>()).initializer("null").build())
+      .addProperty(PropertySpec.builder("prop3", typeNameOf<Callable<String>?>()).initializer("null").build())
+      .addProperty(PropertySpec.builder("prop4", typeNameOf<Function<Int, Int>?>()).initializer("null").build())
+      .addKotlinDefaultImports()
+      .addDefaultPackageImport("java.util.function")
+      .build()
+    assertThat(spec.toString()).isEqualTo(
+      """
+      |import java.util.concurrent.Callable
+      |
+      |val prop0: String? = null
+      |val prop1: Int? = null
+      |val prop2: Map<String, Any>? = null
+      |val prop3: @FunctionalInterface Callable<String>? = null
```

Comment:
Oh wow, where do these annotations come from?

Reply:
Seeeemingly from our typeNameOf. I guess these are included on the resolved KType and we should prolly ignore them